### ISSUE TITLE
学習の再開

### ISSUE THREAD

**Issue by pcoge** [pcoge/AnimeCNN#4]
## 目的
学習済みモデル（hdf5形式）から学習の再開をできるようにする。
466Epochまで連続学習させると、以下のエラーが発生する。
そのため、300Epochまで学習済みモデルを利用して、精度を向上させる。

> Buffered data was truncated after reaching the output size limit

## TaskList
+ [ ] 学習済みモデル（hdf5）の読み込み・学習の再開させる。
+ [ ] 3000Epochまで学習させる。
+ [ ] loss, acc, val_loss, val_accの推移をグラフ化する。

## Doneの定義
TaskListの全解決。

## 参考
Epoch 466 → loss: 0.6322 - acc: 0.8047 - val_loss: 0.5653 - val_acc: 0.8281
> Kerasのノウハウ覚え書き
> https://qiita.com/ak11/items/67118e11b756b0ee83a5

## 留意事項
+ 

## 検討事項
+
